# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/svg/SVGTransformableElement.h
# Commit: b848ca90888e
# Full Hash: b848ca90888ee8e446f3d54b65c3ef68c71a1451
# Author: longsonr <longsonr@gmail.com>
# Date: 2019-06-20 22:06:31
# Regressor Bug: 1551030
# File Overlap Count: 2
# Description:
#   Bug 1551030 - Merge all SVGRect classes r=dholbert
# ==============================================================================

diff -r 20956a783e08 -r b848ca90888e dom/svg/SVGTransformableElement.h
--- a/dom/svg/SVGTransformableElement.h	Thu Jun 20 14:19:57 2019 +0300
+++ b/dom/svg/SVGTransformableElement.h	Thu Jun 20 07:03:54 2019 -0700
@@ -20,7 +20,7 @@
 class DOMSVGAnimatedTransformList;
 class SVGGraphicsElement;
 class SVGMatrix;
-class SVGIRect;
+class SVGRect;
 struct SVGBoundingBoxOptions;
 
 class SVGTransformableElement : public SVGElement {
@@ -36,8 +36,8 @@
   SVGElement* GetNearestViewportElement();
   SVGElement* GetFarthestViewportElement();
   MOZ_CAN_RUN_SCRIPT
-  already_AddRefed<SVGIRect> GetBBox(const SVGBoundingBoxOptions& aOptions,
-                                     ErrorResult& rv);
+  already_AddRefed<SVGRect> GetBBox(const SVGBoundingBoxOptions& aOptions,
+                                    ErrorResult& rv);
   already_AddRefed<SVGMatrix> GetCTM();
   already_AddRefed<SVGMatrix> GetScreenCTM();
   already_AddRefed<SVGMatrix> GetTransformToElement(